src/components/ui/OakRadioGroup/OakRadioGroup.tsx

```diff
@@ -36,13 +46,17 @@ export const OakRadioGroup = (props: OakRadioGroupProps) => {
     $font = "body-1-bold",
     $gap = "space-between-s",
     disabled,
+    value,
+    defaultValue = "",
```

Comment:
Adding these two props allow us to set the initial value for the group (using `defaultValue`) or to make the entire group "controlled" where the internal state can be fully managed by React. This matches the props of a regular `<input />` in Reactland. 

I'm using this in Storybook, but it's a useful pattern to adopt.